stackrox-container-image-scanner/api.yaml

```diff
@@ -10618,15 +11731,27 @@ components:
           title: We never need both sets of layers so consolidate them. They will be ordered by oldest->newest
         dataSource:
           $ref: '#/components/schemas/storageDataSource'
+        version:
+          type: string
+          format: uint64
+          title: Version is used to determine if the metadata needs to be re-pulled
+      title: |-
+        If any fields of ImageMetadata are modified including subfields, please check pkg/images/enricher/metadata.go
+        to ensure that those changes will be automatically picked up
+        Next Tag: 6
     storageImageNote:
       type: string
       enum:
         - MISSING_METADATA
         - MISSING_SCAN_DATA
+        - MISSING_SIGNATURE
+        - MISSING_SIGNATURE_VERIFICATION_DATA
```

Comment:
When this values are returned? Does user need to manually change something in configuration or it's happens always after upgrade?

Reply:
Maybe we should use https://openapi-generator.tech/docs/generators/java/#:~:text=enumUnknownDefaultCase

Reply:
Discussed this offline with @janisz , we agreed on creating a new task that will deal with this option and potentially more: [ROX-11370](https://issues.redhat.com/browse/ROX-11370).